packages/fast-components/src/slider-label/slider-label.ts

```diff
@@ -0,0 +1,58 @@
+import { attr, FastElement, observable } from "@microsoft/fast-element";
+import { FASTSlider } from "../slider";
+import { SliderOrientation } from "../slider/slider";
+
+export class SliderLabel extends FastElement {
+    @observable
+    public positionStyle: string;
+
+    @observable
+    public root: HTMLDivElement;
+
+    @attr
+    public max;
+
+    @attr
+    public position: string;
+
+    @attr
+    public label: string;
+
+    @attr({ attribute: "hide-mark", mode: "boolean" })
+    public hideMark: boolean = false;
```

Comment:
what happens if this changes after instantiation?

Reply:
If you are talking about position that has been fixed, if you are talking about hideMark I'm not sure if anything needs to be done.